swagger_server/itm/itm_scenario.py

```diff
@@ -188,6 +201,16 @@ def change_scene(self, next_scene_id):
             else:
                 # No characters were specified in the scene, so inherit characters from previous scene.
                 target_state.characters = previous_scene_characters
+            # if removed_characters found in scene, remove those characters from the
+            if hasattr(self.isd.current_scene, 'removed_characters'):
```

Comment:
Won't this always be true because you initialized this in `itm_scenario_reader` to `[]` if it's missing?  In theory, you should be able to check its length instead, but defensive programming suggests you should check for `None` also, so:
```
if getattr(self.isd.current_scene, 'removed_characters', None) and len(self.isd.current_scene.removed_characters) > 0:
```